lib/semantics/tools.cc

```diff
@@ -443,6 +490,45 @@ bool IsAssumedSizeArray(const Symbol &symbol) {
   return details && details->IsAssumedSize();
 }
 
+bool IsExternalInPureContext(const Symbol &symbol, const Scope &scope) {
+  if (const auto *pureProc{semantics::FindPureProcedureContaining(&scope)}) {
+    if (const Symbol * root{GetAssociationRoot(symbol)}) {
```

Comment:
Did you run clang format on this? 
I think there should be no space between `*`  and `root`

Reply:
There should be no space, but it is a problem with clang-format. It gets it right with `const auto` but not if there is a type name there.

Reply:
I see. You can ignore my comment Pete.

Reply:
Thanks for checking, though.